deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -0,0 +1,388 @@
+Transform: 'AWS::Serverless-2016-10-31'
+Description: >
+  - Creates a series of lambdas and a step function to process manifests.
+  - The lambdas create iiif manifest and pyramid tiffs out of images.
+  - The finished product are copied to a bucket to serve manifests out of and an image server
+    to display the iiif images.
+  - Creates a bucket for to serve manifests out of
+  - Creates a bucket that the step functions use as an in process location.
+  - Requires the iiif-service and mellon-data-broker pipelines to be built.  Uses ssm keys from those services
+Parameters:
+  EnvType:
+    Type: String
+    Default: 'test'
+  ImageSourceBucket:
+    Type: 'AWS::SSM::Parameter::Value<String>'
+    Description: The name of the source bucket that the IIIF Image service will read from for its assets.
+    Default: '/all/stacks/mellon-data-broker/publicbucket'
+  ImageServerUrl:
+    Type: 'AWS::SSM::Parameter::Value<String>'
+    Description: The url of the IIIF Image Server
+    Default: '/all/stacks/mellon-image-service/url'
+  InfrastructureStackName:
+    Type: String
+    Default: "mellon-app-infrastructure"
+    Description: The name of the parent infrastructure/networking stack that you created. Necessary
+                 to locate and reference resources created by that stack.
+
+Outputs:
+  StateMachine:
+    Value: !Ref StateMachine
+
+  ProcessBucket:
+    Value: !Ref ProcessBucket
+
+  ManifestBucket:
+    Value: !Ref ManifestBucket
+
+Resources:
+
+  CanvasDefaultWidth:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/canvas-default-width"
+      Value: "2000"
+      Description: Canvas default width
+
+  CanvasDefaultHeight:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/canvas-default-height"
+      Value: "2000"
+      Description: Canvas default height
+
+  ImageServerBaseUrl:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/image-server-base-url"
+      Value: "https://STUB:8182/iiif/2"
+      Description: Image server base url
+
+  MainCsv:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/main-csv"
+      Value: "main.csv"
+      Description: CSV name
+
+  SequenceCsv:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/sequence-csv"
+      Value: "sequence.csv"
+      Description: CSV name
+
+  ManifestServerBaseUrl:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/manifest-server-base-url"
+      Value: "https://"
```

Comment:
Is this correct? iirc - this was causing some issues with a couple of the step functions where the parameter passed had a double "https://https://" prefix for the URL